kolibri/plugins/facility/assets/src/views/users/NewUsersPage.vue

```diff
@@ -216,8 +224,68 @@
         noNewUsersLabel$,
         addNewUserLabel$,
         noNewUsersDescription$,
+        currentUserId,
+        isSuperuser,
+        isAdmin,
       };
     },
+    computed: {
+      hasSelectedUsers() {
+        return this.selectedUsers && this.selectedUsers.size > 0;
+      },
+      listContainsLoggedInUser() {
+        return this.selectedUsers.has(this.currentUserId);
+      },
+      canAssignCoaches() {
+        if (!this.hasSelectedUsers) return false;
+        return this.facilityUsers
+          .filter(user => this.selectedUsers.has(user.id))
+          .every(
+            user =>
+              user.kind.includes(UserKinds.COACH) ||
+              user.kind === UserKinds.ADMIN ||
+              user.kind === UserKinds.SUPERUSER ||
+              user.is_superuser,
+          );
+      },
+      canEnrollOrRemoveFromClass() {
+        if (!this.hasSelectedUsers) return false;
+        return this.facilityUsers
+          .filter(user => this.selectedUsers.has(user.id))
+          .every(
+            user =>
+              user.kind === UserKinds.LEARNER ||
+              user.kind.includes(UserKinds.COACH) ||
+              user.kind === UserKinds.ADMIN ||
+              user.kind === UserKinds.SUPERUSER ||
+              user.is_superuser,
+          );
+      },
+      hasSelectedSuperusers() {
+        if (!this.hasSelectedUsers || !this.facilityUsers) return false;
+
+        return this.facilityUsers
+          .filter(user => this.selectedUsers.has(user.id))
+          .some(user => {
+            const isSuperuser =
+              user.kind === UserKinds.SUPERUSER ||
+              user.kind === 'superuser' ||
```

Comment:
Did you run into an issue that a `user.kind` could be something other than the value of `UserKinds.SUPERUSER`?